web/src/main/resources/assets/js/main-template.js

```diff
@@ -34,11 +34,6 @@ if (ghenv.environment === 'development') {
     var autocomplete = AutoComplete.prototype.createStub();
 } else {
     var autocomplete = new AutoComplete(ghenv.geocoding.host, ghenv.geocoding.api_key);
-    // overwrite default for production
-    GHRequest.prototype.hasTCSupport = function() {
-       if(this.api_params.turn_costs !== false)
-          this.api_params.turn_costs = new Set(["car", "truck", "small_truck", "scooter"]).has(this.api_params.vehicle);
-    };
```

Comment:
this can probably be done server-side (using the header telling us the request comes from the client)